src/viam/components/board/board.py

```diff
@@ -30,45 +30,55 @@ class Board(ComponentBase):
         RESOURCE_NAMESPACE_RDK, RESOURCE_TYPE_COMPONENT, "board"
     )
 
-    class AnalogReader:
+    class Analog:
         """
-        AnalogReader represents an analog pin reader that resides on a Board.
+        AnalogReader represents an analog pin reader or writer that resides on a Board.
         """
 
         name: str
-        """The name of the analog reader"""
+        """The name of the analog pin"""
 
         def __init__(self, name: str):
             self.name = name
 
         @abc.abstractmethod
         async def read(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> int:
             """
-            Read the current value.
+            Read the current value from a reader.
```

Comment:
[nit] `from THE reader` rather than `from A reader`, since this method is on a specific instance